### ISSUE TITLE
Error when setting lower soft limit

### ISSUE THREAD

**Issue by DougCoffland** [buildbotics/bbctrl-firmware#249]
When working from the local monitor, an error sometimes comes up when trying to set the lower soft limit. The error says:
Level: error
Source: Comm
Location: src/command.c:200
Repeat: 1
Message: Failed to parse float $0tn=-

Note, in this case I was trying to set the lower soft limit to -200 on Motor 0, which is the x axis. If I clear the error and refresh the page, the value is set back to 0. Also, this doesn't happen every time, but once it starts, I can get out of it by connecting across a network and then the value gets set correctly. When the error occurs, the value gets set to 0 even though it was previously set to -200.

**Comment by tmadrms** [comment]
Thank you for opening this!  Sorry, I got distracted once I found the work around (setting it from remote computer).

**Comment by jcoffland** [comment]
The error is saying that it tried the parse the value ``-`` which is not a valid number.

**Comment by DougCoffland** [comment]
Another customer raised this issue again, so I did a bit more checking. He pointed out that the failure does not occur when using a remote connection.

I found that I can successfully set the soft limit on my local head when using the xontab soft keyboard with my ASUS VT168H touch screen monitor.

However, when I try to set the soft limit to a negative number using a hardware keyboard, I get the error code and failure every time.
